changelog/_ 1686.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+kmip (enterprise): Use entropy augmentation to generate kmip cryptographic information
```

Comment:
Lets be a touch more specific that it's the kmip certificates, and not other stuff.

Reply:
Done!